### PR TITLE
Breadcrumbs

### PR DESCRIPTION
**Pull request by sastels**
make a breadcrumb component

Example on the second form (How did that Happen?)

### CODE REVIEW

**Comment by sastels** [comment]
[Create breadcrumb component](https://trello.com/c/CCBA2YD9/146-create-breadcrumb-component)

File: frontend/src/components/breadcrumb/presets.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+export const Breadcrumb = props => {
+  var paddingValue
+
+  paddingValue = 0
+
+  return (
+    <BreadcrumbList
+      fontSize={[1, null, 2]}
+      lineHeight={[1, null, 2]}
+      pl={paddingValue}
```
Comment (evadb) [inline frontend/src/components/breadcrumb/presets.js]: is passing a paddingValue necessary? It looks like it never changes from 0?

File: frontend/src/components/breadcrumb/presets.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      mb={4}
+      css={css`
```
Comment (evadb) [inline frontend/src/components/breadcrumb/presets.js]: Since list-style-type: none is set in the component itself, I think this can be removed

**Comment by evadb** [comment]
image

Mobile margin looks a bit big if the breadcrumbs end up on multiple lines

File: frontend/src/components/breadcrumb/presets.js (hunk 1 above)
Comment (sastels) [reply frontend/src/components/breadcrumb/presets.js]: hrm, what the heck is `pl` anyway? I stole that code from `ordered-list` but what is `pl` there?

**Comment by sastels** [comment]
[Build basic components](https://trello.com/c/2j6bZwVU/178-build-basic-components)

**Pull request merged**